docs/submit-opentargets-batch.md

```diff
@@ -1,25 +1,44 @@
-# How to submit an OpenTargets batch
-Please follow the instructions below in order to create and submit an OpenTargets batch using ClinVar data. Additional diagrams and background explanations can be found in [this presentation](https://docs.google.com/presentation/d/1nai1dvtfow4RkolyITcymXAsQqEwPJ8pUPcgjLDCntM).
+# How to submit an Open Targets batch
+This protocol describes how to create and submit an Open Targets batch using ClinVar data. Additional diagrams and background explanations can be found in [this presentation](https://docs.google.com/presentation/d/1nai1dvtfow4RkolyITcymXAsQqEwPJ8pUPcgjLDCntM).
 
-Before starting the process, follow the [Build instructions](build.md). In particular, you'll need to install Python 3.5 (if you don't have it already), build the Java ClinVar parser, build and install the Python pipeline.
+Batch submission process consists of seven major parts. For each step, create a single JIRA ticket. This can be done by cloning a corresponding ticket from one of the previous batches and adjusting its parameters. List of suitable ticket templates for each of the steps:
+1. https://www.ebi.ac.uk/panda/jira/browse/EVA-1774
+2. https://www.ebi.ac.uk/panda/jira/browse/EVA-1775
+3. https://www.ebi.ac.uk/panda/jira/browse/EVA-1776
+4. https://www.ebi.ac.uk/panda/jira/browse/EVA-1777
+5. https://www.ebi.ac.uk/panda/jira/browse/EVA-1778
+6. https://www.ebi.ac.uk/panda/jira/browse/EVA-1779
+7. https://www.ebi.ac.uk/panda/jira/browse/EVA-1780
```

Comment:
May be my attention span to scan documents isn't what it used to be but is this preferable to having the template links beside the individual sections?

Reply:
I see your point, and originally the template links _were_ scattered throughout the document, each near the header line of the corresponding section. However, this turned out to be not convenient in real life, because the _only_ time you want to look at those templates is when creating the new tickets for the current batch. And this is done simultaneously for all issues, at the very start of processing the new batch.